src/User32.Desktop/User32.cs

```diff
@@ -1534,7 +1554,7 @@ public static partial class User32
         ///         <see cref="Win32ErrorCode.ERROR_NOT_ENOUGH_QUOTA" /> when the message limit is hit.
         ///     </para>
         /// </returns>
-        [DllImport(nameof(User32), SetLastError = true)]
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
Probably it would make sense to also change from `IntPtr` to `void*` and have the code generation offer the `IntPtr` version here.